eureka2-core/src/main/java/com/netflix/eureka2/utils/rx/PauseableSubject.java

```diff
@@ -0,0 +1,169 @@
+package com.netflix.eureka2.utils.rx;
```

Comment:
It's hard to diff here, is this just a renamed NotificationsSubject (nice name change btw) or are there code changes as well?